client-bagel-buddy/src/components/header/Header.js

```diff
@@ -50,6 +52,16 @@ function Header() {
         }
     };
 
+    const updateBio = (text) => {
```

Comment:
remove text as we are not referencing it anywhere
const updateBio = () =>

Reply:
(to level up)
when everything works without error, you may want to replace the `alert` in the 
`if (res.success)` to
```
setUser(res.data);
setIsEdit(false);
```
the first line set the user to the new user with updated bio
the second one turns off the edit mode